lib/generators/templates/uploadcare_config_template.erb

```diff
@@ -0,0 +1,129 @@
+  # because your users will still be able to choose any file via drag&drop or from URL.
+  # config.input_accept_types = 'image/*'
+
+  # Defines the list of preferred MIME types.
+  # The list should be ordered and contain space-separated MIME types.
+  # Common parts can be marked with asterisks: image/* application/vnd.openxmlformats-officedocument.*.
+  # If no MIME types match the criteria or preferred types aren’t set, default formats are used.
+  # Keep in mind that some cloud services can export data in different formats.
+  # For example, Google document can be exported as Word document, PDF or plain text.
+  # config.preferred_types = 'image/* application/vnd.openxmlformats-officedocument.*'
+
+  # Forces a system-native file picking dialog to show up instead of our file uploader.
+  # That makes the file uploader behavior as close as possible to the generic <input type="file">.
+  # Native behavior is achieved at the expense of support for uploading content
+  # from social media and cloud storage, manual crop, and preview step.
+  # Multi-file selection would still work.
+  # The option does not work in old browser versions: the dialog falls back to the file uploader.
+  # config.system_dialog = false
+
+  # This option sets the file size threshold for multipart uploading.
+  # The value ranges from 10485760 (10 MB), which is default, up to 104857600 (100 MB).
+  # If a file size hits the threshold, it gets uploaded in four parallel chunks by 5 MB.
+  # Multipart upload makes large file uploading faster.
+  # Files go directly to a storage, bypassing our upload servers,
+  # and they're quickly available for further use. All files below the threshold get uploaded in one piece.
+  # Note: Multipart upload works with local sources only (files and camera).
+  # config.multipart_min_size = 10485760
+
+  # Available locales currently are:
+  # ar az ca cs da de el en es et fr he it ja ko lv nb nl pl pt ro ru sk sr sv tr uk vi zhTW zh
+  config.locale = 'en'
+
+  # Sets custom localization options. See https://uploadcare.com/docs/uploads/file-uploader/#localization
+  # config.locale_translations = {}
+
+  # Defines pluralization options. See https://uploadcare.com/docs/uploads/file-uploader/#pluralization
+  # config.locale_pluralize = {}
+
+  # The signature is an HMAC/SHA256 hex-encoded hash that includes secret_key and expire strings.
+  # To use Secure Uploading for your project, you need to make a signature on your back end using your
+  # project's Secret Key and provide it to File Uploader.
+  # Learn how to make your signature in docs - https://uploadcare.com/docs/security/secure-uploads/
+  # config.secure_signature = ''
+
+  # Stands for the Unix time to which the signature is valid, e.g., 1454902434.
+  # Works together with the secure signature option that defines a period of your signature validity.
+  # config.secure_expire = 1454902434
```

Comment:
this definitely does not look like a proper conf setting
it should be lifetime, not expire.